stacks.yml

```diff
@@ -529,7 +529,7 @@ lambda_logs_subscription:
     - cloudwatch_logs_lambda
   parameters:
     Lambda: "{{ stacks.cloudwatch_logs_lambda.outputs.Arn }}"
-    FilterPattern: ''
+    FilterPattern: '{ $.request != "*/_status?ignore-dependencies *" || $.url != "*/_status?ignore-dependencies" }'
```

Comment:
We might want to expand this to all `/_status` requests?  These will all generally be the "Apps are up" smoketests.  Not sure if there's anything else we can obviously throw away at this stage... probably keep it all for now.